app/views/stories/_sign_in_invitation.html.erb

```diff
@@ -1,19 +1,22 @@
 <div class="single-article single-article-small-pic feed-cta" id="in-feed-cta">
   <div class="cta-container" id="cta-content">
+    <img class="rainbowdevimage" src="<%= asset_path "rainbowdev.svg" %>" /> 
     <h2>
-      <% if @tag.blank? %>
-        Need more relevant posts?<br />Sign in to customize your feed:
-      <% else %>
-        <a href="/">dev.to</a> is a community of <br /><%= number_with_delimiter User.count %> amazing humans who code.
-      <% end %>
+      <a href="/">DEV</a> is a community of <br /><%= number_with_delimiter User.count %> amazing humans who code.
```

Comment:
Do we want `User.count` for the live numbers?  I feel like `User.size` is better for this case even though it's less up-to-date.

Reply:
Huh? I'm not sure quite what you mean.

Reply:
Ah never mind; I thought there was a `size` method on `User` but turns out there isn't.